build/package/scripts/build-npm.sh

```diff
@@ -68,9 +68,10 @@ fi
 echo "Configuring npm to use Nexus (${NEXUS_URL}) ..."
 if [ -n "${NEXUS_URL}" ] && [ -n "${NEXUS_USERNAME}" ] && [ -n "${NEXUS_PASSWORD}" ]; then
     NEXUS_AUTH="$(urlencode "${NEXUS_USERNAME}"):$(urlencode "${NEXUS_PASSWORD}")"
+    NEXUS_HOST=$(echo "${NEXUS_URL}" | sed 's/http[s]\?:\/\///')
```

Comment:
Maybe we can align this https://github.com/opendevstack/ods-pipeline/blob/master/build/package/scripts/build-python.sh#L60?